### DISCUSSION TITLE
arduino-multi-relay2

### DISCUSSION THREAD

**Discussion by szybki946** [lkankowski/arduino-multi-relay2#37] [Q&A]
Nie wiem czemu ale program pokazuje błąd przy:
const uint8_t MONO_STABLE_TRIGGER = LOW;

const char MULTI_RELAY_DESCRIPTION[] PROGMEM = "Multi Relay";

const RelayConfigDef gRelayConfig[] PROGMEM = {

const ButtonConfigDef gButtonConfig[] PROGMEM = {

**Comment by lkankowski** [comment]
Ciekawe jaki to błąd i jaka cała konfiguracja :)

**Comment by szybki946** [comment]
Witam mam problem, wszystko poprawie ustawione w programie. A1 zapala 2 światła.
{A0,  MONO_STABLE, 1,  -1, -1, "SWIATLO PRZED DOMEM_1"},


zapala 2 światła:
 {1,  2,  RELAY_TRIGGER_HIGH, "SWIATLO PRZED DOMEM_1"},
 {13, 14, RELAY_TRIGGER_HIGH, "SWIATLO SYPIALNIA"},

[dom test.zip](https://github.com/lkankowski/arduino-multi-relay2/files/9227117/dom.test.zip)

Post poprawiony. Pisałem dlatego tak bo, nie wiedziałem czy pisać tutaj w Pl czy EN. Niestety nie znam angielskiego. Myślę że teraz oczy cię nie bolą.
Czytałem twój post i sobie myślę co on pisze. Przeglądarka czasem próbuje tłumaczyć, wtedy wychodzą głupoty.

**Reply by lkankowski** [reply]
Wybacz, że przyczepię się jednej kwestii zanim przejdę do sedna - przeczytałem twój wpis chyba z 5 razy i dalej nie jestem pewny, czy dobrze rozumiem.
Musisz mieć świadomość, że interpunkcja, ortografia, "nie" pisane razem lub osobno, ogólna stylistyka, itp. niektórych tak rozpraszają, że nie są w stanie skupić się na sednie. Ja najwyraźniej należę do takich osób i dlatego apel z mojej strony, żeby szanować czas innych i skorzystać z dobrodziejstw sprawdzania pisowni i maksymalnie jasno opisywać problem.
I mam nadzieję bez urazy - to taka informacja zwrotna, która w życiu może się przydać ;)

Co do sedna problemu, to nie widzę niestety żadnych błędów w konfiguracji, więc możliwe są jakieś problemy w połączeniach elektrycznych. Sprawdzałeś Arduino na płytce stykowej?

**Reply by lkankowski** [reply]
A przepraszam - masz jeden błąd - pin "A1" powtarza się 2 razy.

**Comment by szybki946** [comment]
Nie wiem czy program testowałeś. Nie wiem czemu tak jest ale A1 zapalało 2 światła ,ale nie tylko A0 był też inny włącznik który zapalał 2 światła. Przestawiłem ID z 1 na 2 i problem znikł, nie wiem czemu tak ale pomogło. Bez urazy doceniam  twoją prace wiedzą że zrobiłeś to bez kosztów.

**Comment by szybki946** [comment]
Prosił bym o usunięcie dyskusji wszystko jest poprawne

**Reply by lkankowski** [reply]
A co było powodem problemów?

**Reply by szybki946** [reply]
w programie były otwarte 2 podobne stechy i podczas wgrywania nakładały się. Więc sorki za zakłócanie czasu.
